Relays_driver_firmware/Relays_driver_firmware.ino

```diff
@@ -0,0 +1,155 @@
+#define emonTxV3                                                                          // Tell emonLib this is the emonTx V3 - don't read Vcc assume Vcc = 3.3V as is always the case on emonTx V3 eliminates bandgap error and need for calibration http://harizanov.com/2013/09/thoughts-on-avr-adc-accuracy/
+#define RF69_COMPAT 1                                                              // Set to 1 if using RFM69CW or 0 is using RFM12B
+#include <JeeLib.h>                                                                      //https://github.com/jcw/jeelib - Tested with JeeLib 3/11/14
+ISR(WDT_vect) { Sleepy::watchdogEvent(); }                            // Attached JeeLib sleep function to Atmega328 watchdog -enables MCU to be put into sleep mode inbetween readings to reduce power consumption 
+#include "EmonLib.h" 
+
+#include "EmonLib.h"  
+typedef struct { 
+  int relay1_status;
+  int relay2_status;
+  int relay3_status;
+  int relay4_status;
+} Playload;     // create structure - a neat way of packaging data for RF comms
+
+Playload load_monitor ;
+
+//-----------------------RFM12B / RFM69CW SETTINGS----------------------------------------------------------------------------------------------------
+byte RF_freq=RF12_433MHZ;                                        // Frequency of RF69CW module can be RF12_433MHZ, RF12_868MHZ or RF12_915MHZ. You should use the one matching the module you have.
+byte nodeID=8;                                                 // emonpi node ID
+uint32_t networkGroup = 210;
+boolean RF_STATUS =1;
+static int intRFtime = 600000;
+// RF Global Variables 
+static byte stack[RF12_MAXDATA+4], top, sendLen, dest;           // RF variables 
+static char cmd;
+static word value;                                               // Used to store serial input
+
+unsigned long last_relay_driver_check = 0;        
+const long check_relays_interval = 2000; 
+
+
+const int relay_pins [] = {9,8,7,6};   // pin number of relays; 
+const byte number_of_relays=4;
+word relays_status [] = {number_of_relays};
+
+void setup()
+{
+  Serial.begin(38400);
```

Comment:
Can you make this baud rate a global constant so it' easier to configure. 
Put it at the top of the file in all caps